### ISSUE TITLE
768*768，4090 24GB VRAM  overload

### ISSUE THREAD

**Issue by isamu2025** [crystallee-ai/controlGIF#2]
<img width="1718" alt="wechat_20231126202936" src="https://github.com/crystallee-ai/controlGIF/assets/1702623/ca1df515-0071-4700-84f4-aa3207785f0c">
<img width="680" alt="wechat_20231126203015" src="https://github.com/crystallee-ai/controlGIF/assets/1702623/2b733761-0976-463b-b0f2-71456868e740">

<img width="289" alt="wechat_20231126203324" src="https://github.com/crystallee-ai/controlGIF/assets/1702623/63f46fff-7314-49e5-827b-475a1ad942cc">

**Comment by crystallee-ai** [comment]
Yes, this problem does exist, I think I will solve it with the development of Stable Diffusion, or directly use the super-resolution model。
